tests/probes_test.go

```diff
@@ -89,6 +89,64 @@ var _ = Describe("[ref_id:1182]Probes", func() {
 			table.Entry("[test_id:1200][posneg:positive]with working HTTP probe and http server", httpProbe, tests.StartHTTPServer),
 		)
 
+		table.DescribeTable("should succeed with 'Host' field", func(readinessProbe *v12.Probe) {
+
+			isHTTPProbe := readinessProbe.Handler.HTTPGet != nil
+			isDualStack := tests.IsIPv6Cluster(virtClient)
```

Comment:
Yeah, this is wrong.

`tests.IsIPv6Cluster` is reading the IP of the DNS service, and returning if it is an IPv6 address. In dual stack, the DNS service returns an IPv4 addr.

Reply:
So we need a new method that iterate over the IPs and check if we have one IPv6 right ?

Reply:
yes, seems that will be needed. I have something like that on #4035 , but tailored for the `Service` entity. 

I'll push something that generalizes that, so you can re-use it. Is that OK ?